styles.css

```
@@ -0,0 +1,196 @@
+    height: 0.5rem;
+    border: 0.5rem solid var(--text-color);
+    top: 65%;
+  }
+}
+@media (min-width: 1023px) {
+  .marker::before {
+    top: 50%;
+    width: 0.5rem;
+    height: 0.5rem;
+    border: 0.5rem solid var(--text-color);
+    top: 70%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

top is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)